pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -203,11 +203,16 @@ def _tk_gate_set(backend: "QiskitBackend") -> Set[OpType]:
     """Set of tket gate types supported by the qiskit backend"""
     config = backend.configuration()
     if config.simulator:
-        return {
+        gate_set = {
             _gate_str_2_optype[gate_str]
             for gate_str in config.basis_gates
             if gate_str in _gate_str_2_optype
-        }.union({OpType.Measure, OpType.Reset, OpType.Barrier})
+        }.union({OpType.Measure, OpType.Reset, OpType.Barrier, OpType.RangePredicate})
```

Comment:
not sure if this is ok...

Reply:
Is the `RangePredicate` a new optype? Not sure I am understanding this correctly?

Reply:
It's not new. Before it was added to the `GateSetPredicate` of the `AerBackend` only (not `AerState/UnitaryBackend`. Some gates supported by the qiskit backends, such as the ones here, don't show up in the backends `.configuration().basis_gates`, so they apparently need to be added manually.

However, I don't know if it always should be these four for any `Aer` backend. Which is what I have configured now. I also don't know how to find out.

Reply:
Also I now add `Unitary1qBox` and `Unitary3qBox` if `unitary` is in the basis gates. Which makes since from what I can tell but not 100% sure.

Reply:
Ok, I think these gates shouldn't be added for the AerState and AerUnitary Backends so I changed that.

Reply:
Okay, looks good!